layouts/partials/news/list/summary.html

```diff
@@ -0,0 +1,28 @@
+<div class="row">
+<div class="col-md-21">
+    <div class="news-content">
+      {{ partial "news/date" . }}
+      {{ $href := .Params.redirect_url | default .RelPermalink }}
+      <div class="content-inner">
+          <span class="flex-column" style="height: 100%;">
+            {{ partial "news/title" . }}
+            {{ if isset .Params "image"}}
+              <a href="{{ $href }}" class="content-link">
+                <img src="{{ .RelPermalink }}{{.Params.image}}" class="content-image" />
+              </a>
+            {{ end }}
+            <p class="content-summary">{{ .Params.summary | truncate 260 }}</p>
+            {{ partial "news/authors" . }}
+            <p class="news-footer">
+              <a href="{{ $href }}" class="content-link read-more">
+                Continue reading
```

Comment:
Jakarta.ee is currently available in English and Chinese language.

Because of that requirement, I would to see the implementation of i18n functions in these partials: https://gohugo.io/functions/i18n/

Reply:
sure will do now

Reply:
done